docker/compose/docker-compose.yml

```diff
@@ -79,8 +79,11 @@ services:
     ports:
       - "12345:12345"
     command: segmentstore
+    privileged: true
```

Comment:
Fixed. Only the NFS version which needs mount inside a container should have it. No other docker compose should have it.